mpt/impl/imap-mailbox/core/src/main/resources/org/apache/james/imap/scripts/Login.test

```diff
@@ -29,8 +29,8 @@ S: a002a BAD LOGIN failed. Illegal arguments.
 C: a003 LOGIN invaliduser password
 S: a003 NO LOGIN failed. Invalid login/password.
 
-C: a004 LOGIN imapuser invalid
+C: a004 LOGIN imapuser@domain invalid
```

Comment:
I mean, maybe you have wrong setting on "virtual users". Try swipping it, you, in my opinion, do not need these changes.